### PR TITLE
Improve releasing during 0.8.2

### PR DESCRIPTION
**Pull request by gerhard**
Thanks to @wingyplus's excellent improvements to the Elixir SDK, we are now able to make it part of our release process. We are now able to produce the following SDK releases at the same time:
1. 🐹 Go
2. 🐍 Python
3. ⬢ Node.js
4. 🧪 Elixir

### CODE REVIEW

**Comment by wingyplus** [comment]
🎉🎉

File: RELEASING.md [hunk 1]
```diff
@@ -350,6 +354,53 @@ gh release create "sdk/nodejs/${NODEJS_SDK_VERSION:?must be set}" \
...
+- [ ] Ensure that all checks are green ✅ for the `<SDK_GIT_SHA>` on the `main`
+  branch that you are about to release. This will usually be the commit that
+  bumps the Engine version, the one that you merged earlier.
+- [ ] Tag & publish:
+
+
+```console
+cd sdk/elixir && export ELIXIR_SDK_VERSION=$(changie latest) && cd ../..
+git tag "sdk/elixir/${ELIXIR_SDK_VERSION:?must be set}" "${SDK_GIT_SHA:?must be set}"
+git push origin sdk/elixir/${NODEJS_SDK_VERSION}
```
Comment (wingyplus) [inline RELEASING.md:378]:
```suggestion
git push origin sdk/elixir/${ELIXIR_SDK_VERSION}
```

**Comment by wingyplus** [comment]
Push a fix for incorrect env name in Elixir.

**Comment by gerhard** [comment]
Thank you @wingyplus!

**Review by grouville — APPROVED** [review]
✅

**Review by wingyplus — APPROVED** [review]

**Pull request merged**